src/components/common/ColonyHome/ColonyTitle/ColonyAddress.css

```diff
@@ -0,0 +1,24 @@
+@value query700 from '~styles/queries.css';
+
+.colonyAddress {
+  padding: 4px 6px;
+  height: 24px;
+  border-radius: var(--radius-normal);
+  text-align: center;
+  line-height: var(--size-smallish);
+  color: var(--grey-purple);
+  cursor: pointer;
+  user-select: none;
+  outline: 0;
+}
+
+.colonyAddress:hover {
+  background-color: color-mod(var(--colony-blue) alpha(10%));
```

Comment:
`color-mod` for me has been erroring since we moved to cdapp -- so not sure why it's fine here. Maybe the :hover which is actually redundant since we're only using `ColonyAddress` on mobile.

FYI this is how to get around it: https://github.com/JoinColony/colonyCDapp/issues/195

Reply:
Right now this appears to work as expected. On mobile if you click on the colony address, the background color appears in the correct shade of colony blue. When tapping elsewhere on screen the background color disappears.